src/asInput/asInput.scss

```diff
@@ -1,3 +1,6 @@
+// TODO: Remove this dependency
```

Comment:
Should this TODO be done for this PR, or is there a ticket for it? Generally I like to avoid leaving TODOs in committed code if possible.

Reply:
I think I could probably delete this whole file honestly. The component itself is deprecated and I'll remove in a PR soon and these styles are doing very little.